framework/auth/oauth_scopes.py

```diff
@@ -62,9 +72,9 @@ class ComposedScopes(object):
                     (CoreScopes.NODE_BASE_WRITE, CoreScopes.NODE_CHILDREN_WRITE, CoreScopes.NODE_LINKS_WRITE)
 
     # Privileges relating to editing content uploaded under that node # TODO: Add wiki etc when implemented
-    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ,)
+    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, CoreScopes.NODE_COMMENTS_READ)
```

Comment:
Including ability to make comments as well?

If that's the case, you'll want to move the comment CoreScopes items to the `NODE_METADATA_READ` composed scope (etc).